app/build.gradle

```diff
@@ -87,7 +88,7 @@ android {
             buildConfigField "boolean", "USE_RELEASE_VERSIONING", "true"
             def deepLinkSchemeValue = "fenix-nightly"
             buildConfigField "String", "DEEP_LINK_SCHEME", "\"$deepLinkSchemeValue\""
-            manifestPlaceholders = ["deepLinkScheme": deepLinkSchemeValue]
+            manifestPlaceholders = ["deepLinkScheme": deepLinkSchemeValue, "requestLegacyExternalStorage": false]
```

Comment:
Should we maybe set 'false' also for debug to allow any new issues related to this be caught in development?

Reply:
I tried to added it to the `debug` version but for some reason the value were not set (It kept reporting `true)`,  not sure if it has do with other variants having  `releaseTemplate >>`, I will give it another try today.

Reply:
Addressed!